spec/amp-story-player.md

```diff
@@ -0,0 +1,201 @@
+---
+$category@: presentation
+formats:
+  - websites
+  - stories
+teaser:
+  text: A player for embedding and playing your favorite stories in your own website.
+---
+
+<!--
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="`amp-story-player`"></a> `amp-story-player`
+
+<figure class="centered-fig">
+  <amp-anim alt="amp-story-player example" width="300" height="533" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/spec/img/amp-story-player.gif">
+    <noscript>
+    <img alt="amp-story-player example" src="https://github.com/ampproject/spec/img/amp-story-player.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>An easy way to embed and play stories in a website.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Scripts</strong></td>
+    <td>
+    <code>&lt;script async src="https://cdn.ampproject.org/amp-story-player-v0.js">&lt;/script></code>
+    <code>&lt;link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel='stylesheet' type='text/css'></code>
+    </td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><ul>
+      <li>See <a href="https://github.com/ampproject/amphtml/blob/master/examples/amp-story/player.html">code snippet</a>.</li>
+    </ul></td>
+  </tr>
+</table>
+
+[TOC]
+
+## AMP story player usage
+
+Use `amp-story-player` to embed and play stories within a webpage.
+
+### Embed in a non-AMP page
+
+The following example demonstrates a basic embed of <amp-story-player> in a non-AMP webpage.
```

Comment:
Wrap `<amp-story-player>` in backticks please

Reply:
Change to:
The code snippet below demonstrates an embed of `<amp-story-player>` in a non-AMP webpage.